service/header_service_test.go

```diff
@@ -385,3 +533,21 @@ func createHeader(height int32, hash chainhash.Hash, prevBlock chainhash.Hash) d
 		Chainwork:     big.NewInt(4295032833),
 	}
 }
+
+func toChainhash(hash string) chainhash.Hash {
```

Comment:
maybe it could be moved next to `fixutres.HashOf` function and named as `HashValueOf`
or you can simply use the HashOf method in the following way: `*fixtures.HashOf("....")` to get a value instead of pointer.